shell/app/modules/cmp/pages/middleware-dashboard/index.tsx

```diff
@@ -153,15 +153,15 @@ const MiddlewareDashboard = () => {
       title: i18n.t('node'),
       dataIndex: 'nodes',
       key: 'nodes',
-      width: '10%',
+      width: 80,
```

Comment:
Is it wide enough in English mode？

Reply:
It's enough to 'node' column, But the 'number of references' column is not wide enough.